File: uikit/src/main/java/io/agora/chat/uikit/chat/interfaces/OnChatRecordTouchListener.java
package io.agora.chat.uikit.chat.interfaces;

import android.view.MotionEvent;
import android.view.View;

public interface OnChatRecordTouchListener {
    /**
     * Record pressing events
     * @param v
     * @param event
     * @return
     */
    boolean onRecordTouch(View v, MotionEvent event);
}

File: ease-im-kit/src/main/kotlin/io/agora/uikit/feature/chat/interfaces/OnChatRecordTouchListener.kt
package io.agora.uikit.feature.chat.interfaces

import android.view.MotionEvent
import android.view.View

interface OnChatRecordTouchListener {
    /**
     * Record pressing events
     * @param v
     * @param event
     * @return
     */
    fun onRecordTouch(v: View?, event: MotionEvent?): Boolean
}